tst/EnergyPlus/unit/CMakeLists.txt

```diff
@@ -68,6 +68,7 @@ set( test_src
   FanCoilUnits.unit.cc
   Fans.unit.cc
   FaultsManager.unit.cc
+  FileSystem.unit.cc
```

Comment:
Yet another new unit test file, we've got to be nearly done with new unit test files.  :+1: